src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -604,12 +604,11 @@ private void ProcessRstStreamFrame(FrameHeader frameHeader)
                 return;
             }
 
-            _incomingBuffer.Discard(frameHeader.Length);
+            var protocolError = (Http2ProtocolErrorCode)BinaryPrimitives.ReadInt32BigEndian(_incomingBuffer.ActiveSpan);
```

Comment:
nit: var

Reply:
This is a correct use of 'var'.  'var' can be used when the type is obvious, i.e. either from a `new` statement or an explicit cast.